R/download_predicate_dsl.R

```diff
@@ -143,17 +155,24 @@
 #' - establishmentMeans (ESTABLISHMENT_MEANS)
 #' - coordinateUncertaintyInMeters (COORDINATE_UNCERTAINTY_IN_METERS)
 #' - gadm (GADM_GID) (for the Database of Global Administrative Areas)
+#' - level0Gid (GADM_LEVEL_0_GID)
+#' - level1Gid (GADM_LEVEL_1_GID)
+#' - level2Gid (GADM_LEVEL_2_GID)
+#' - level3Gid (GADM_LEVEL_3_GID)
 #' - stateProvince (STATE_PROVINCE)
 #' - occurrenceStatus (OCCURRENCE_STATUS)
 #' - publishingOrg (PUBLISHING_ORG)
-#' - occurrenceId (OCCURRENCE_ID)
-#' - eventId (EVENT_ID)
+#' - occurrenceID (OCCURRENCE_ID)
+#' - eventID (EVENT_ID)
```

Comment:
I don't agree with *Id -> ID. 

I think it should be the same as it was, since that would be an unnecessary breaking change.

Reply:
Why can't we support both?

Reply:
Oh good point! Yes support both.

Reply:
Updated